packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyscanner/backpack-react-scripts",
-  "version": "9.1.0",
+  "version": "9.1.1",
```

Comment:
Can you revert this? We bump this version as part of the release process already so this causes issues with that, and blocks CI correctly from running

Reply:
Yeah, Have restored it to `v9.1.0`.